### DISCUSSION TITLE
playcover3.0 运行新石器时代一直进不去，以下是报错信息

### DISCUSSION THREAD

**Discussion by Superuser-fank** [PlayCover/PlayCover#1025] [General]
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               cn [3052]
Path:                  /Users/USER/Library/Containers/io.playcover.PlayCover/新石器时代.app/cn
Identifier:            com.giraffe.game.stoneage
Version:               1.0.8 (155)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2023-07-03 07:52:20.3998 +0800
OS Version:            macOS 13.4.1 (22F82)
Report Version:        12
Anonymous UUID:        C5443CC5-A109-AF4C-1541-41DEEF64670D

Sleep/Wake UUID:       EF65A840-F486-4C7E-80CC-0257D0C7B82D

Time Awake Since Boot: 7400 seconds
Time Since Wake:       484 seconds

System Integrity Protection: enabled

Crashed Thread:        4  Dispatch queue: CAImageQueue::server_reply_queue

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Application Specific Information:
abort() called

**Comment by TheMoonThatRises** [comment]
Please open this as an issue, not a discussion so it is easier to track.
